### PR TITLE
[PATRO23-56] Local keycloak environment

### PR DESCRIPTION
**Pull request by DawidBski**
https://tracker.intive.com/jira/browse/PATRO23-56

### CODE REVIEW

**Review by AlanP22 — APPROVED** [review]

File: docker-compose.override.yml [hunk 1]
```diff
@@ -7,16 +7,37 @@ services:
...
   intive.patronage2023.api:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
       - ASPNETCORE_URLS=https://+:443;http://+:80
       - ASPNETCORE_Kestrel__Certificates__Default__Password=awesomepass
       - ASPNETCORE_Kestrel__Certificates__Default__Path=/https/intivepatronageapi.pfx
     ports:
-      - "8080:80"
-      - "8081:443"
+      - "5080:80"
```
Comment (mateuszmuszynski) [inline docker-compose.override.yml:20]: Any reason why it has been changed?

File: src/api/app/Program.cs [hunk 2]
```diff
@@ -6,6 +6,7 @@
 using Intive.Patronage2023.Shared.Infrastructure.EventDispachers;
 using Intive.Patronage2023.Shared.Infrastructure.EventHandlers;
 using Microsoft.AspNetCore.HttpLogging;
+using Microsoft.EntityFrameworkCore;
```
Comment (mateuszmuszynski) [inline src/api/app/Program.cs:9]: It's not related to keycloak

File: src/modules/example/api/ExampleModule.cs [hunk 3]
```diff
@@ -22,7 +22,7 @@ public static class ExampleModule
 	/// <returns>Updated IServiceCollection.</returns>
 	public static IServiceCollection AddExampleModule(this IServiceCollection services, ConfigurationManager configurationManager)
 	{
-		services.AddDbContext<ExampleDbContext>(options => options.UseSqlServer(configurationManager.GetConnectionString("AppDb")));
+		services.AddDbContext<ExampleDbContext>(options => options.UseSqlServer(configurationManager.GetConnectionString("DockerDb")));
```
Comment (mateuszmuszynski) [inline src/modules/example/api/ExampleModule.cs:25]: Not related to KC, create another PR

**Review by mateuszmuszynski — CHANGES_REQUESTED** [review]

File: docker-compose.override.yml (hunk 1 above)
Comment (DawidBski) [reply docker-compose.override.yml:20]: Meybe my reason is weak and stupid a little but i think ports 8080 and 8081 are more related to  [running keycloak in a container instruction](https://www.keycloak.org/server/containers#_building_the_docker_image).

File: src/api/app/Program.cs (hunk 2 above)
Comment (DawidBski) [reply src/api/app/Program.cs]: I've already corrected it.

File: docker-compose.override.yml (hunk 1 above)
Comment (mateuszmuszynski) [reply docker-compose.override.yml:20]: Good point. Let's leave it as it is.

File: src/modules/example/api/ExampleModule.cs (hunk 3 above)
Comment (DawidBski) [reply src/modules/example/api/ExampleModule.cs]: I've changed it back.

**Review by mateuszmuszynski — APPROVED** [review]

**Pull request merged**